orderer/common/multichannel/util_test.go

```diff
@@ -31,6 +31,10 @@ func (c *mockChainCluster) StatusReport() (types.ClusterRelation, types.Status)
 	return types.ClusterRelationMember, types.StatusActive
 }
 
+func (c *mockChainCluster) HaltExternal() {
+	c.mockChain.Halt()
+}
+
```

Comment:
Why is this needed?

Reply:
Good catch. This is leftover from the previous implementation in the halt PR.  I'll remove this.